packages/core/src/app/coupon/components/Discounts.tsx

```diff
@@ -1,25 +1,40 @@
 import React, { type FunctionComponent, useState } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
 
 import { TranslatedString } from '@bigcommerce/checkout/locale';
 import { IconCoupon, IconDownArrow, IconUpArrow } from '@bigcommerce/checkout/ui';
 
 import { ShopperCurrency } from '../../currency';
 import { type DiscountItem, useMultiCoupon } from '../useMultiCoupon';
 
-const DiscountItem: FunctionComponent<{ coupons: DiscountItem[] }> = ({ coupons }) => {
-    return coupons.map((coupon) => (
-        <div
-            aria-live="polite"
-            className="cart-priceItem optimizedCheckout-contentPrimary"
-            key={coupon.name}
-        >
-            <span className="cart-priceItem-label"><IconCoupon />{coupon.name}</span>
-            <span className="cart-priceItem-value">
-                -<ShopperCurrency amount={coupon.amount} />
-            </span>
-        </div>
-    ));
-}
+const DiscountItems: FunctionComponent<{ coupons: DiscountItem[] }> = ({ coupons }) => {
+    return (
+        <TransitionGroup component={null}>
+            {coupons.map((coupon) => {
+                const nodeRef = React.createRef<HTMLDivElement>();
+                return (
+                    <CSSTransition
+                        classNames="changeHighlight"
+                        key={coupon.name}
+                        nodeRef={nodeRef}
+                        timeout={500}
```

Comment:
I see we are using `timeout={{}}`  now.
https://github.com/bigcommerce/checkout-js/blob/83e366f51cf2f1e4ec54157a40604b0d75cab678/packages/core/src/app/checkout/CheckoutStep.tsx#L190
Shall we keep things in line?

Reply:
I can get rid of it actually, looking at other implementations.